routes/screeningmovies.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express';
+import { fetchScreenings } from '../utils/screeningUtils.js';
+import { mapScreenings } from '../utils/screeningUtils.js';
+import { frontpageScreening } from '../utils/screeningUtils.js';
+// import { fetchScreenings } from '../utils/screeningUtils.js';
+const screeningRouter = express.Router();
+
+
+
+
+
+
```

Comment:
I would clean out all of this extra whitespace. One or two empty lines can be a good way to separate code, but after that it's difficult to keep things consistent and better to not go beyond two empty lines (usually one is enough).

Automatic formatters like [Prettier](https://prettier.io) can help with this.